nemo_text_processing/text_normalization/taggers/cardinal.py

```diff
@@ -30,10 +31,14 @@ class CardinalFst(GraphFst):
     """
     Finite state transducer for classifying cardinals, e.g. 
         -23 -> cardinal { negative: "true"  integer: "twenty three" } }
+
+    Args:
+        deterministic: if True will provide a single transduction option,
+            for False multiple transduction are generated (used for audio-based normalization)
     """
 
-    def __init__(self):
-        super().__init__(name="cardinal", kind="classify")
+    def __init__(self, deterministic: bool):
```

Comment:
you may set the default to True for all classes

Reply:
done